test/functional/test_framework/test_framework.py

```diff
@@ -619,6 +619,26 @@ def join_network(self):
         self.connect_nodes(1, 2)
         self.sync_all()
 
+    def generate(self, generator, nblocks, **kwargs):
+        """
+        Genarate nblocks blocks.
```

Comment:
in commit 45817a7239cd5c0ef9126fbdf33bc72283eafa3a: Typo.

Also the signature can now be generalized to `*args, **kwargs`?

Reply:
no ragrets